nncf/common/graph/graph_matching.py

```diff
@@ -94,6 +108,14 @@ def __add__(self, other):
 
 
 class AlternatingExpression(Expression):
+    """A composite expression that matches to any of the node path in the graph which matches to at least
```

Comment:
Please, format the docstring according to the Style Guide.

Reply:
Done